src/twister2/specification_processor.py

```diff
@@ -222,6 +227,12 @@ def should_skip_for_toolchain(test_spec: YamlTestSpecification, platform: Platfo
     if test_spec.toolchain_exclude and test_spec.toolchain_exclude & set(platform.toolchain):
         _log_test_skip(test_spec, platform, 'platform.toolchain in testcase.toolchain_exclude')
         return True
+    if (used_toolchain_version not in platform.toolchain) \
+            and ('host' not in platform.toolchain) \
+            and (test_spec.type != 'unit'):
+        msg = f'platform.toolchain not supported by currently used toolchain "{used_toolchain_version}"'
```

Comment:
message string can be passed directly to `log_test_skip`, it is not used in any other places

Reply:
I left it as a separate variable just because this string is quite long and if I move it directly to `log_test_skip`, then I need to brake this function call into several lines. So all in all for me is more readable to keep it as separate variable.